cves/CVE-2007-3847.yml

```diff
@@ -120,8 +127,10 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-    - commit:
-      note:
+    - commit: 13bdffd40b879b52adc435e299411f6d48190d62
+      note: |
+        Simply put, I believe the vcc performed a complex bound check to protect
```

Comment:
A simplified explanation of the bound check, perhaps an edge case, might be interesting to read! Perhaps we can learn to look for similar cases in the future?